docs/outdated/index.rst

```diff
@@ -0,0 +1,25 @@
+**********************
+Outdated Documentation
+**********************
+
+The following pages are no longer relevant to the most recent release of TARDIS:
```

Comment:
Specify the release number

Reply:
Since that is always changing, I can direct them to where to find the latest release if that works